src/models/gc9a01.rs

```diff
@@ -0,0 +1,171 @@
+
+        dcs.write_raw(0x90, &[0x08, 0x08, 0x08, 0x08])?;
+        dcs.write_raw(0xBD, &[0x06])?;
+        dcs.write_raw(0xBC, &[0x00])?;
+        dcs.write_raw(0xFF, &[0x60, 0x01, 0x04])?;
+
+        dcs.write_raw(0xC3, &[0x13])?; // power control 2
+        dcs.write_raw(0xC4, &[0x13])?; // power control 3
+        dcs.write_raw(0xC9, &[0x22])?; // power control 4
+
+        dcs.write_raw(0xBE, &[0x11])?;
+        dcs.write_raw(0xE1, &[0x10, 0x0E])?;
+        dcs.write_raw(0xDF, &[0x20, 0x0c, 0x02])?;
+
+        dcs.write_raw(0xF0, &[0x45, 0x09, 0x08, 0x08, 0x26, 0x2A])?; // gamma 1
+        dcs.write_raw(0xF1, &[0x43, 0x70, 0x72, 0x36, 0x37, 0x6f])?; // gamma 2
+        dcs.write_raw(0xF2, &[0x45, 0x09, 0x08, 0x08, 0x26, 0x2A])?; // gamma 3
+        dcs.write_raw(0xF3, &[0x43, 0x70, 0x72, 0x36, 0x37, 0x6f])?; // gamma 4
+
+        dcs.write_raw(0xED, &[0x18, 0x0B])?;
+        dcs.write_raw(0xAE, &[0x77])?;
+        dcs.write_raw(0xCD, &[0x63])?;
+
+        dcs.write_raw(
+            0x70,
+            &[0x07, 0x07, 0x04, 0x0E, 0x0F, 0x09, 0x07, 0x08, 0x03],
+        )?;
+
+        dcs.write_raw(0xE8, &[0x34])?; // framerate
+
+        dcs.write_raw(
+            0x62,
+            &[
+                0x18, 0x0D, 0x71, 0xED, 0x70, 0x70, 0x18, 0x0F, 0x71, 0xEF, 0x70, 0x70,
+            ],
+        )?;
+        dcs.write_raw(
+            0x63,
+            &[
+                0x18, 0x11, 0x71, 0xF1, 0x70, 0x70, 0x18, 0x13, 0x71, 0xF3, 0x70, 0x70,
+            ],
+        )?;
+        dcs.write_raw(0x64, &[0x28, 0x29, 0xF1, 0x01, 0xF1, 0x00, 0x07])?;
+        dcs.write_raw(
+            0x66,
+            &[0x3C, 0x00, 0xCD, 0x67, 0x45, 0x45, 0x10, 0x00, 0x00, 0x00],
+        )?;
+        dcs.write_raw(
+            0x67,
+            &[0x00, 0x3C, 0x00, 0x00, 0x00, 0x01, 0x54, 0x10, 0x32, 0x98],
+        )?;
+
+        dcs.write_raw(0x74, &[0x10, 0x85, 0x80, 0x00, 0x00, 0x4E, 0x00])?;
+        dcs.write_raw(0x98, &[0x3e, 0x07])?;
+        dcs.write_raw(0x35, &[0x00])?; // tearing effect line ON
+
+        dcs.write_command(SetInvertMode(options.invert_colors))?; // set color inversion
+
+        dcs.write_command(ExitSleepMode)?; // turn off sleep
+        delay.delay_us(120_000);
+
+        dcs.write_command(SetDisplayOn)?; // turn on display
+
+        Ok(madctl)
+    }
+
+    fn write_pixels<DI, I>(&mut self, dcs: &mut Dcs<DI>, colors: I) -> Result<(), Error>
+    where
+        DI: WriteOnlyDataCommand,
+        I: IntoIterator<Item = Self::ColorFormat>,
+    {
+        dcs.write_command(WriteMemoryStart)?;
+        let mut iter = colors.into_iter().map(|c| c.into_storage());
+
+        let buf = DataFormat::U16BEIter(&mut iter);
+        dcs.di.send_data(buf)?;
+        Ok(())
+    }
+
+    fn default_options() -> ModelOptions {
+        let mut options = ModelOptions::with_sizes((240, 240), (240, 240));
+        options.set_invert_colors(ColorInversion::Inverted);
```

Comment:
@almindor Should we remove this now or later? We had planned to remove the default inversions in issue #48.

Reply:
Yah I think new models should be set without inversions.

Reply:
done